### PR TITLE
update angular to rc.4

### PR DESCRIPTION
**Pull request by alemata**

### CODE REVIEW

File: 01-Login/app/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+23 error If you do, this is most likely a problem with the angular2-quickstart package,
+23 error not with npm itself.
+23 error Tell the author that this fails on your system:
+23 error     tsc && concurrently "tsc -w" "lite-server"
+23 error You can get information on how to open an issue for this project with:
+23 error     npm bugs angular2-quickstart
+23 error Or if that isn't available, you can get their info via:
+23 error     npm owner ls angular2-quickstart
+23 error There is likely additional logging output above.
+24 verbose exit [ 1, true ]
```
Comment (flaviogranero) [inline 01-Login/app/npm-debug.log]: let's exclude that file

File: 02-Custom-Login/npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+23 error If you do, this is most likely a problem with the angular2-quickstart package,
+23 error not with npm itself.
+23 error Tell the author that this fails on your system:
+23 error     tsc && concurrently "tsc -w" "lite-server"
+23 error You can get information on how to open an issue for this project with:
+23 error     npm bugs angular2-quickstart
+23 error Or if that isn't available, you can get their info via:
+23 error     npm owner ls angular2-quickstart
+23 error There is likely additional logging output above.
+24 verbose exit [ 1, true ]
```
Comment (flaviogranero) [inline 02-Custom-Login/npm-debug.log]: let's exclude this one too

**Comment by alemata** [comment]
@flaviogranero could you review please?

**Comment by flaviogranero** [comment]
👍 @ntotten

**Pull request merged**